testsuite/tests/kuadrant/authorino/identity/test_redirect.py

```diff
@@ -0,0 +1,40 @@
+"""
+Test for authorino redirect
+"""
+# pylint: disable=unused-argument
+import pytest
+
+from testsuite.openshift.objects.auth_config import AuthConfig
+
+STATUS_CODE = 302
+REDIRECT_URL = 'http://anything.inavlid?redirect_to='
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, authorino, envoy, blame, openshift, module_label, oidc_provider):
+    """In case of Authorino, AuthConfig used for authorization"""
+    if authorization is None:
+        authorization = AuthConfig.create_instance(openshift, blame("ac"),
+                                                   envoy.hostname, labels={"testRun": module_label})
+    authorization.add_oidc_identity("rhsso", oidc_provider.well_known["issuer"])
```

Comment:
I would move this test to `authorino/` dir only and in that case this will be redundant